docs/user/resources/01-serverless.md

```diff
@@ -0,0 +1,89 @@
+     conditions:
+     - lastTransitionTime: "2023-04-28T10:09:37Z"
+       message: Configured with default Publisher Proxy URL and default Trace Collector
+         URL.
+       reason: Configured
+       status: "True"
+       type: Configured
+     - lastTransitionTime: "2023-04-28T10:15:15Z"
+       message: Serverless installed
+       reason: Installed
+       status: "True"
+       type: Installed
+     eventPublisherProxyURL: http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish
+     state: Ready
+     traceCollectorURL: http://telemetry-otlp-traces.kyma-system.svc.cluster.local:4318/v1/traces
+   ```
+
+## Custom resource parameters
+<!-- TABLE-START -->
+### Serverless.operator.kyma-project.io/v1alpha1
+
+**Spec:**
+
+| Parameter | Type | Description |
+| ---- | ----------- | ---- |
+| **dockerRegistry**  | object |  |
+| **dockerRegistry.&#x200b;enableInternal**  | boolean |  |
+| **dockerRegistry.&#x200b;secretName**  | string |  |
+| **eventing**  | object |  |
+| **eventing.&#x200b;endpoint** (required) | string |  |
+| **tracing**  | object |  |
+| **tracing.&#x200b;endpoint** (required) | string |  |
+
+**Status:**
+
+| Parameter | Type | Description |
+| ---- | ----------- | ---- |
+| **conditions**  | \[\]object | Conditions associated with CustomStatus. |
+| **conditions.&#x200b;lastTransitionTime** (required) | string | lastTransitionTime is the last time the condition transitioned from one status to another. This should be when the underlying condition changed.  If that is not known, then using the time when the API field changed is acceptable. |
+| **conditions.&#x200b;message** (required) | string | message is a human readable message indicating details about the transition. This may be an empty string. |
+| **conditions.&#x200b;observedGeneration**  | integer | observedGeneration represents the .metadata.generation that the condition was set based upon. For instance, if .metadata.generation is currently 12, but the .status.conditions[x].observedGeneration is 9, the condition is out of date with respect to the current state of the instance. |
+| **conditions.&#x200b;reason** (required) | string | reason contains a programmatic identifier indicating the reason for the condition's last transition. Producers of specific condition types may define expected values and meanings for this field, and whether the values are considered a guaranteed API. The value should be a CamelCase string. This field may not be empty. |
+| **conditions.&#x200b;status** (required) | string | status of the condition, one of True, False, Unknown. |
+| **conditions.&#x200b;type** (required) | string | type of condition in CamelCase or in foo.example.com/CamelCase. --- Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be useful (see .node.status.conditions), the ability to deconflict is important. The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt) |
```

Comment:
```suggestion
| **conditions.&#x200b;type** (required) | string | type of condition in camelCase or in `foo.example.com/CamelCase`. --- Many **.condition.type** values are consistent across resources like `Available`, but because arbitrary conditions can be useful (see **.node.status.conditions**), the ability to deconflict is important. The regex it matches is `(dns1123SubdomainFmt/)?(qualifiedNameFmt)` |
```